src/map.rs

```diff
@@ -3117,6 +3138,7 @@ fn capacity() {
 #[cfg(test)]
 mod tests {
     use super::*;
+
```

Comment:
This and the other new newlines below also seem like unrelated changes?

Reply:
It seems like my IDE formatter added and then cargo fmt decides to keep it.